opaque_keys/edx/tests/test_locators.py

```diff
@@ -59,3 +61,44 @@ def test_version_tree(self):
         test_id = ObjectId(test_id_loc)
         valid_locator = CourseLocator(version_guid=test_id)
         self.assertEqual(VersionTree(valid_locator).children, [])
+
+
+@ddt.ddt
+class AggregateCourseLocatorTests(TestCase):
+    """
+    Tests for :class:`.AggregateCourseLocator`
+    """
+
+    def test_aggregate_course_locator(self):
+        aggregate_course_key = 'aggregate-course:org+course'
+        aggregate_course_locator = AggregateCourseLocator(org='org', course='course')
+        self.assertEqual(aggregate_course_locator, AggregateCourseKey.from_string(aggregate_course_key))
+
+    @ddt.data(
+        'org/course/run',
+        'course-v1:org+course+run',
+    )
+    def test_aggregate_course_locator_from_course_key(self, course_id):
+        course = CourseKey.from_string(course_id)
+        aggregate_course_locator = AggregateCourseLocator.from_course_key(course)
+
+        aggregate_course_key = 'aggregate-course:{}+{}'.format(course.org, course.course)
+        self.assertEqual(aggregate_course_locator, AggregateCourseKey.from_string(aggregate_course_key))
+
+    def test_aggregate_course_locator_serialize(self):
+        aggregate_course_key = 'aggregate-course:org+course'
+        aggregate_course_locator = AggregateCourseKey.from_string(aggregate_course_key)
```

Comment:
You should not need to use `AggregateCourseKey.from_string` to instantiate a new instance of `AggregateCourseKey`. Fix this here, and in all other tests where it is used unnecessarily.